### ISSUE TITLE
Migrate All Seach Queries in Frontend to GQL V2

### ISSUE THREAD

**Issue by SudharakaP** [opencollective/opencollective#5377]
As per https://github.com/opencollective/opencollective-api/pull/7294 we support the `search` query in GQL v2. There were three places were we use this in the front-end. We need to migrate all these to GQL v2. 

- [x] [`search.js`](https://github.com/opencollective/opencollective-frontend/blob/7a22582838dcf9e17e73a88ddaa2fb9748022df6/pages/search.js#L270); Done at, https://github.com/opencollective/opencollective/issues/5364
- [ ] [`CollectivePickerAsync.js`](https://github.com/opencollective/opencollective-frontend/blob/a9694ae464bd2457b70a764d948c2167e99b30d8/components/CollectivePickerAsync.js#L12)
- [ ] [`UnhostAccountForm.js`](https://github.com/opencollective/opencollective-frontend/blob/a9694ae464bd2457b70a764d948c2167e99b30d8/components/CollectivePickerAsync.js#L12)

**Comment by znarf** [comment]
To be clear, as part of your current project, it's not asked to migrate everything. Just what you are working on.

**Comment by SudharakaP** [comment]
> To be clear, as part of your current project, it's not asked to migrate everything. Just what you are working on.

Yeah, this is just for future. The other issue tracks what is relevant to the search experience redesign project; https://github.com/opencollective/opencollective/issues/5364
